habitat/sims/habitat_simulator.py

```diff
@@ -208,18 +197,23 @@ def create_sim_config(
         agent_config.sensor_specifications = sensor_specifications
         agent_config.action_space = {
             SimulatorActions.LEFT.value: habitat_sim.ActionSpec(
-                "lookLeft", {"amount": self.config.TURN_ANGLE}
+                "turn_left",
+                habitat_sim.ActuationSpec(amount=self.config.TURN_ANGLE),
```

Comment:
What exactly is the use for `ActuationSpec`, is it to factor in noise?

Reply:
Current the ActuationSpec just specifies how much the action moves the agent.  Be it the number of degrees to turn or the number of meters to move forward.  I made it a separate class that way actions that have more things to configure (noisy actions) will fit into the same paradigm, but with their own subclass of `ActuationSpec`.